src/handlers/setDependencyVersionHandler.ts

```diff
@@ -0,0 +1,198 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as fse from "fs-extra";
+import * as vscode from "vscode";
+import { mavenExplorerProvider } from "../explorer/mavenExplorerProvider";
+import { Dependency } from "../explorer/model/Dependency";
+import { MavenProject } from "../explorer/model/MavenProject";
+import { constructDependenciesNode, constructDependencyNode, getIndentation } from "../utils/editUtils";
+import { UserError } from "../utils/errorUtils";
+import { ElementNode, getNodesByTag, XmlTagName } from "../utils/lexerUtils";
+import { getVersions } from "../utils/requestUtils";
+
+const CONFLICT_INDICATOR: string = "omitted for conflict";
+
+export async function setDependencyVersionHandler(selectedItem?: Dependency): Promise<void> {
+    if (selectedItem === undefined) {
+        throw new UserError("No dependency node specified.");
+    }
+    const pomPath: string = selectedItem.projectPomPath;
+    if (!await fse.pathExists(pomPath)) {
+        throw new UserError("Specified POM file does not exist on file system.");
+    }
+
+    let effectiveVersion: string;
+    if (selectedItem.supplement.indexOf(CONFLICT_INDICATOR) !== -1) {
```

Comment:
Better do this in parsing stage.  Replace `supplement` with an optional data structure, e.g.: 
omitted?: {
reason: "conflict" | "duplicate";
effectiveVersion: string | undefined;
}

You can search gloablly whether you have multiple definition of `CONFLICT_INDICATOR` and calls of `indexOf(CONFLICT_INDICATOR)`?